tools/HitboxExpander/hitbox_expander.py

```diff
@@ -65,8 +65,8 @@ def add(x, y):
       pix[coords] = (0, 0, 0, 1)
 
     PngSave(im, path)
-  #except:
-  #  print("Could not process " + name)
+  except:
+    print("Could not process " + name)
```

Comment:
```suggestion
  except Exception as e:
    print("Could not process " + name)
    print(e)
```

No reason not to post the exception too.